rootfs/etc/nginx/lua/lua_ingress.lua

```diff
@@ -2,25 +2,40 @@ local _M = {}
 
 local seeds = {}
 local original_randomseed = math.randomseed
-math.randomseed = function(seed)
-  local pid = ngx.worker.pid()
 
-  if seeds[pid] then
+local function get_seed_from_urandom()
+  local seed
+  local frandom = io.open("/dev/urandom", "rb")
+  if frandom then
+    local str = frandom:read(4)
```

Comment:
Please make sure you handle errors in `:read` and `.open` calls.

Reply:
added error info, and the `:read` will return nil without err if cannot read data.